e2e-tests/portal/cypress/support/utils/keyboardInput.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * keyboardInput
+ * Clear and type text into an input.
+ * @param {Function} selector: Cypress selector
+ * @param {String} text: Text to enter
+ */
+const keyboardInput = (selector, text) => {
```

Comment:
in EXIP, we have the root E2E commands folder, could we have similar here? means this only needs to be declared in one place and not copied to each folder

Reply:
Would be good to do this separately, lots of commands are duplicated in DTFS, for example `saveSession`, `login`.